javascript/organizing_your_javascript_code/factory_functions_and_module_pattern.md

```diff
@@ -263,17 +263,18 @@ Take the calculator example into consideration. It's very easy to imagine a scen
 <div class="lesson-content__panel" markdown="1">
 
 1. WesBos has a beautiful and in-depth section on scopes and closures. Please check out these sections under "Module 3 - The Tricky Bits":
-  - [The article on scope](https://wesbos.com/javascript/03-the-tricky-bits/scope)
-  - [The article on closures](https://wesbos.com/javascript/03-the-tricky-bits/closures)
+-    [The article on scope](https://wesbos.com/javascript/03-the-tricky-bits/scope)
+-    [The article on closures](https://wesbos.com/javascript/03-the-tricky-bits/closures)
```

Comment:
```suggestion
   - [The article on scope](https://wesbos.com/javascript/03-the-tricky-bits/scope)
   - [The article on closures](https://wesbos.com/javascript/03-the-tricky-bits/closures)
```
What I meant was for the unordered list itself to be indented 3 spaces, as opposed to putting more spaces between the `-` and their text.

Reply:
![image](https://github.com/TheOdinProject/curriculum/assets/122839503/967892dd-ad95-454e-a3d0-0635336b678d)

Particularly for little changes when there aren't that many of them, you can hit the "commit suggestion" button to auto-commit a review suggestion rather than having to do it manually.
Note that this will put your local out of sync, so you'd have to do something like `git fetch origin descriptive-link-text` followed by `git merge origin/descriptive-link-text` if you wanted to update your local branch with any of these changes made in the GitHub UI.

Reply:
Oh sorry about that! Thank you for letting me know about the GitHub UI feature. The indentation should be fixed now.